libfwupdplugin/fu-plugin.c

```diff
@@ -611,30 +611,12 @@ gboolean
 fu_plugin_has_custom_flag(FuPlugin *self, const gchar *flag)
 {
 	FuPluginPrivate *priv = GET_PRIVATE(self);
-	GPtrArray *guids;
 
 	g_return_val_if_fail(FU_IS_PLUGIN(self), FALSE);
 	g_return_val_if_fail(flag != NULL, FALSE);
 
 	/* never set up, e.g. in tests */
```

Comment:
I know I'm late to this party, but the comments seems irrelevant for the code now (and raises the question if tests should've been handled differently in the new change too)